src/drive/lib/RealTimeQueries.jsx

```diff
@@ -1,12 +1,15 @@
 import { memo, useEffect } from 'react'
-import { useClient, Mutations } from 'cozy-client'
+import { useClient, Mutations, models } from 'cozy-client'
```

Comment:
File global remark: At this point if we think that it's useful we can move this file to cozy-client ?

Reply:
I'll create a PR in cozy-client.